documentation/getting-started.md

```diff
@@ -1,64 +1,29 @@
 # Get started
 
-## Step 1: Install the barista-components and Angular CDK
+## TLDR;
 
-Install the Barista components library via npm or yarn:
+`ng add @dynatrace/barista-components`
 
-`npm install --save @dynatrace/barista-components @angular/cdk highcharts@6 @types/highcharts @dynatrace/barista-icons d3-scale@^3.0.0 d3-shape@^1.3.5`  
-or  
-`yarn add @dynatrace/barista-components @angular/cdk highcharts@6 @types/highcharts @dynatrace/barista-icons d3-scale@^3.0.0 d3-shape@^1.3.5`
+## Installation
 
-## Step 2: Animations
+To install our barista components use our ng-add schematics with
+`ng add @dynatrace/barista-components`. This will guide you through our
+installation process and will ask you some questions.
 
-Some Barista components depend on the Angular animations module. If you want
-these animations to work in your app, you have to install the
-`@angular/animations` module and include the `BrowserAnimationsModule` in your
-app.
+1. The path to the NgModule where the imports will be registered.
+2. If the Browser animation should be enabled.
```

Comment:
Many component readme files (e.g. confirmation-dialog, expandable-panel/section, table,....) reference "Step 2: Animations" in the getting started guide and link to this section (`https://barista.dynatrace.com/components/get-started/#step-2-animations`). 

As this section will be gone after this PR is merged, we have to remove the links and maybe also the whole sentence as there are not  more details about it anymore.

Reply:
I will handle that in a separate branch.